ONSdigital/response-operations-ui
# Fix secure message return to inbox

# What and why?
Return to the correct inbox was not working correctly for some technical and misc messages when there is recorded survey ID in the secure_message table. The code has been updated so that all technical and misc messages will return to the correct inbox.

# How to test?

1. Run unit tests
2. Deploy this PR to your dev env and run the acceptance tests
3. Create a technical message and misc message via Frontstage
4. In rOps got to navigate to the 2 messages and click on them
5. Click the return button and you should be redirected to the correct inbox

# Jira
https://jira.ons.gov.uk/browse/RAS-1224
